docs/contributors/mocking.md

```diff
@@ -0,0 +1,15 @@
+# Mocking code in operator-registry
```

Comment:
ah, I was going off this:

https://github.com/operator-framework/operator-registry/tree/master/pkg/apprclient/mock

but now I realize that that's probably there because we never cleaned it up? I can look into this and see if I can do it quickly. We probably want to have all of the mocks in the project on the same framework at some point